package.json

```diff
@@ -25,6 +25,7 @@
     "grunt-contrib-uglify": ">=0.5.0",
     "grunt-contrib-jshint": ">=0.10.0",
     "grunt-contrib-jasmine": ">=0.8.2",
+    "grunt-contrib-connect": "~0.3.0",
```

Comment:
Please use `npm install --save-dev grunt-contrib-connect` to place the dependency in the correct place (sorted alphabetically).